### PR TITLE
Make cors accept origin configurable

### LINKED ISSUE #1670: Make Access-Control-Allow-Origin configurable
### Actual behavior

`Access-Control-Allow-Origin` HTTP header is currently set to `*` and doesn't allow user to configure/restrict CORS it for security purposes.

### Expected behavior

User should be able to restrict `Access-Control-Allow-Origin` HTTP header

### Steps to reproduce

- Check https://github.com/LiskHQ/lisk-service/blob/v0.7.0-beta.1/services/gateway/app.js#L112

### Which version(s) does this affect? (Environment, OS, etc...)

v0.7.0-beta.1

### Additional Information

- Allow user to restrict `Access-Control-Allow-Origin` HTTP header
- Ref: https://moleculer.services/docs/0.12/moleculer-web.html#CORS-headers
- Update the swagger and API documentation with the environment variable information

### PR DESCRIPTION
**Pull request by priojeetpriyom**
### What was the problem?
This PR resolves #1670 

### How was it solved?
- [x] Make CORS accept origin configurable

### How was it tested?
- [x] Local

### CODE REVIEW

File: services/gateway/config.js [hunk 1]
```diff
@@ -111,4 +111,8 @@ config.job = {
 	},
 };
 
+config.cors = {
+	allowedOrigin: '*',
```
Comment (nagdahimanshu) [inline services/gateway/config.js:115]: I think we also have to allow users to configure the value:
```suggestion
	allowedOrigin: process.env.CORS_ALLOWED_ORIGIN || '*',
```

File: services/gateway/config.js [hunk 2]
```diff
@@ -111,4 +111,8 @@ config.job = {
 	},
 };
 
+config.cors = {
+	allowedOrigin: process.env.CORS_ALLOWED_ORIGIN || '*',
```
Comment (vardan10) [inline services/gateway/config.js:115]: Please add description of env variable over here:
https://github.com/LiskHQ/lisk-service/blob/development/services/gateway/README.md

File: services/gateway/app.js [hunk 3]
```diff
@@ -109,7 +109,7 @@ tempApp.run().then(async () => {
 			use: [],
 
 			cors: {
-				origin: '*',
+				origin: config.cors.allowedOrigin,
```
Comment (vardan10) [inline services/gateway/app.js:112]: We should be able to support multiple origins.
Something like what is done here https://github.com/moleculerjs/moleculer-web/issues/43
```suggestion
				origin: config.cors.allowedOrigin.split(","),
```

File: services/gateway/config.js (hunk 2 above)
Comment (priojeetpriyom) [reply services/gateway/config.js]: Applied.

File: services/gateway/app.js (hunk 3 above)
Comment (priojeetpriyom) [reply services/gateway/app.js:112]: Applied.

**Review by nagdahimanshu — APPROVED** [review]
LGTM 👍

**Review by vardan10 — APPROVED** [review]

**Pull request merged**